src/features/newPartnerNetwork/partnerNetworkApiSlice.ts

```diff
@@ -45,7 +45,7 @@ export const apiSlice = createApi({
       BusinessPartnerRequest
     >({
       query: (body) => ({
-        url: '/legal-entities/search?page=0&size=30',
+        url: '/members/legal-entities/search?page=0&size=30',
```

Comment:
I tested it and this doesn't solve the issue... certainly also this [query](https://github.com/eclipse-tractusx/portal-frontend/blob/e5a6f82043edf5f46aa7f04a7f2f5137a4133ad0/src/features/newPartnerNetwork/partnerNetworkApiSlice.ts#L57) needs to get updated in some way - only adding /members won't be sufficient as this only returns a 404.
As you are enabled to connect to the int env, you can test this interface change very easily, please make sure to properly test fix before re-requesting the review.

Reply:
https://business-partners.int.catena-x.net/pool/ui/swagger-ui/index.html#/Legal%20Entity%20Controller/searchLegalEntities

Look at this swagger link, only POST request has /members in the URL. GET request has not changed.

I could not test it from my end as it is giving 403(permission issue).

Reply:
As discussed today, the 403 error is why the interface change is needed in the first place, the permission issue will be solved once the interface has been adjusted.

Reply:
Is that change request addressed now? Has the fix been tested?

Reply:
@oyo no, @manojava-gk is still working on the fix